src/main/java/tr/com/orioninc/laborant/service/AdminService.java

```diff
@@ -40,24 +46,26 @@ public  String addNewLab(String labName, String userName, String password, Strin
 
     public List<Lab> getAllLabs(){
         List<Lab> allLabs = labRepo.findAll();
+        log.info("[getAllLabs] - Getting all labs from the database... \n {}", allLabs);
         return allLabs;
-
     }
 
     public String deleteLab(String labName){
         Lab labToBeDeleted = labRepo.findByLabName(labName);
         if (Objects.isNull(labToBeDeleted)){
+            log.warn("[deleteLab] There is no lab named {} in the database to be deleted", labName);
             return "There isn't a lab named "+labName+" found in the database to be deleted";
         }
         else {
             labRepo.delete(labToBeDeleted);
-
+            log.info("[deleteLab] Successfully deleted lab named {}", labName);
             return "Lab named "+labName+" is successfully deleted from the database";
 
         }
     }
 
     public Lab findLabByName(String labName){
+        log.info("[findLabByName] Lab found: {}",labName);
```

Comment:
Lab is not found yet. We can't be sure `labRepo.findByLabName(labName);` always runs successfully.

Reply:
did changed on it